packages/indexer-client/src/dataMappers.ts

```diff
@@ -303,10 +305,22 @@ export function mapIndexerLeaderboardPosition(
     percentRoi: toBigDecimal(position.roi),
     roiRank: toBigDecimal(position.roi_rank),
     accountValue: toBigDecimal(position.account_value),
+    volume: position.volume ? toBigDecimal(position.volume) : undefined,
     updateTime: toBigDecimal(position.update_time),
   };
 }
 
+export function mapIndexerTieredLeaderboardPosition(
+  tieredPosition: IndexerServerTieredLeaderboardPosition,
+): IndexerTieredLeaderboardParticipant {
+  return Object.fromEntries(
```

Comment:
We could just use `mapValues` inline [here](https://github.com/vertex-protocol/vertex-typescript-sdk/pull/230/files#diff-1a59e593d5688c43202632d4464fd753b47ef4b4d3abf2e0593265a8877f146fR838)

Reply:
oh ic ic sg